ecommerce/ecommerce/settings.py

```diff
@@ -31,6 +31,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'products.apps.ProductsConfig',
```

Comment:
Nice, but it's best to add your apps after Django's built in apps. This is because apps further down in the list take precedence over higher up apps, i.e. they override upper apps. Having your app at the top could cause unexpected side effects if one of Django's apps decares a method or class with the same name as yours. Read this for some more info
https://stackoverflow.com/questions/31925458/importance-of-apps-orders-in-installed-apps

Reply:
Noted. Thats just where the tutorial had put the app so i just put it there too but thats good to know.